aopy/preproc/base.py

```diff
@@ -179,15 +179,17 @@ def validate_measurements(expected_values, measured_values, diff_thr):
     corrected_values[diff_above_thr] = expected_values[diff_above_thr]
     return corrected_values, diff_above_thr
 
-def interp_timestamps2timeseries(timestamps, timestamp_values, samplerate=None, sampling_points=None, interp_kind='linear', extrap_values='extrapolate'):
+def interp_timestamps2timeseries(timestamps, timestamp_values, samplerate=None, sampling_points=None, interp_kind='linear', extrapolate=True):
```

Comment:
I think the default value for extrapolate should be False because it is an extension of what the function is designed to do. Extrapolation may happen that users are unaware of and could impact analysis/interpretations.

Reply:
Yeah I agree, was more trying to keep it consistent with the previous implementation